docs/extras/integrations/llms/opaqueprompts.ipynb

```diff
@@ -172,15 +172,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import langchain.utilities.promptguard as pgf\n",
+    "import langchain.utilities.opaqueprompts as op\n",
```

Comment:
why don't use this directly
`from langchain.utilities.opaqueprompts import desanitize`

and L193 `op.desanitize..` -> `desanitize...`

Reply:
We could. `import as` offers a slightly better namespace I think.

Reply:
It is just about a sideeffect. If we use `import langchain.utilities.promptguard as pgf` the doc build does not include the `desanitize` in the` API Reference` section. Here is an example https://python.langchain.com/docs/integrations/providers/ainetwork

Reply:
Oh I see. Didn't notice that. Yes, I can change it that way. Thanks for the insights!